content/docs/hello-world.md

```diff
@@ -15,36 +15,33 @@ ReactDOM.render(
+>Itong gabay ay dinesenyo para sa mga taong mas gusto **pag-aralan ang mga konsepto ng bai-baitang**. Kapag mas gusto mo matuto sa pamamagitan ng paggawa, pwede mong tignan ang aming [praktikal na tutorial](/tutorial/tutorial.html). Iyong mapapansin na ang gabay na ito at ang tutorial ay nagko-komplementaryo sa isa't isa.
 
-This is the first chapter in a step-by-step guide about main React concepts. You can find a list of all its chapters in the navigation sidebar. If you're reading this from a mobile device, you can access the navigation by pressing the button in the bottom right corner of your screen.
+Ito ang unang kabanata sa bai-baitang na gabay patungkol sa pinakamahalagang mga konsepto ng React. Ang mga listahan ng mga kabanata ay makikita sa navigation sidebar. Kung ikaw ay nagbabasa nito gamit ang iyong mobile device, ma-aaccess mo ang navigation sa pamamagitan ng pagpindot sa pindutan sa kanang sulok sa ibaba ng iyong screen.
 
-Every chapter in this guide builds on the knowledge introduced in earlier chapters. **You can learn most of React by reading the “Main Concepts” guide chapters in the order they appear in the sidebar.** For example, [“Introducing JSX”](/docs/introducing-jsx.html) is the next chapter after this one.
+Ang bawat kabanata sa gabay na ito ay karagdagang aral kasabay sa naunang mga kabanata. **Maari mong malaman ang kabuuan ng React sa pamamagitan ng pagbasa sa mga kabanata ng "Main Concepts" ayon sa pagkakasunod-sunod nito sa sidebar.** Halimbawa, [“Introducing JSX”](/docs/introducing-jsx.html) ang kasunod na kabanata pagkatapos nito.
 
-## Knowledge Level Assumptions {#knowledge-level-assumptions}
+## Pagtukoy sa Antas ng Kaalaman {#knowledge-level-assumptions}
 
-React is a JavaScript library, and so we'll assume you have a basic understanding of the JavaScript language. **If you don't feel very confident, we recommend [going through a JavaScript tutorial](https://developer.mozilla.org/en-US/docs/Web/JavaScript/A_re-introduction_to_JavaScript) to check your knowledge level** and enable you to follow along this guide without getting lost. It might take you between 30 minutes and an hour, but as a result you won't have to feel like you're learning both React and JavaScript at the same time.
+Ang React ay isang JavaScript library kung kaya't ina-assume namin na may sapat kang kaalaman sa lengguwaheng JavaScript. **Kung may agam-agam at pagdududa sa iyong sarili, inirerekumenda namin na [sumubok munang mag-aral ng isang JavaScript tutorial](https://developer.mozilla.org/en-US/docs/Web/JavaScript/A_re-introduction_to_JavaScript) para masuri ang antas ng iyong kaalaman** at makasunod sa gabay na ito nang hindi naguguluhan. Maari kang abutin ng mula sa 30 minuto hanggang sa isang oras, ngunit bilang resulta hindi mo mararamdaman na nag-aaral ka ng React at JavaScript nang magkasabay.
```

Comment:
Ang React ay isang JavaScript library kung kaya't ~~ina-assume~~ aming ina-asume ~~namin~~ na may sapat kang kaalaman sa ~~lengguwaheng~~ JavaScript. **Kung may agam-agam at pagdududa ~~sa iyong sarili~~ ka sa iyong nalalaman, inirerekumenda namin na [sumubok munang mag-aral ng isang JavaScript tutorial](https://developer.mozilla.org/en-US/docs/Web/JavaScript/A_re-introduction_to_JavaScript) para masuri ang antas ng iyong kaalaman** at **nang** makasunod sa gabay na ito nang hindi naguguluhan. Maari kang abutin ng mula sa 30 minuto hanggang sa isang oras, ngunit bilang resulta hindi mo mararamdaman na nag-aaral ka ng React at JavaScript nang ~~magkasabay~~ sabay.

Reply:
Amma change this `"naming [sumubukan mo munang magbasa"` to this instead `"naming [subukan mo munang magbasa"`